nb_sb_global.go

```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright (c) 2017 eBay Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package goovn
+
+import (
+	"fmt"
+
+	"github.com/ebay/libovsdb"
+)
+
+type NBGlobalTableRow struct {
```

Comment:
These are types defined mostly for future use. They can be used to get back the entire row in the future.

Reply:
@hzhou8 : Can confirm as in the past we had a discussion on this to have each go file for each entity wherein nb_global and sb_global has to be different go file to be consistent with current code.

Reply:
@hzhou8 any preferences on splitting the implementation into two files?